test/integration/dvl.cc

```diff
@@ -342,6 +343,8 @@ TEST_P(DopplerVelocityLogTest, BottomTrackingWhileStatic)
       4 * config.trackingNoise));
   }
   EXPECT_EQ(0, message.status());
+
+  this->manager.Remove(sensor->Id());
```

Comment:
Should deleting the sensors be something the destructor of `sensors::Manager` should handle instead of requiring the user to do it?

Reply:
Added a call to clear sensors in manager destructor. Segfault still happens when exiting test so kept the `Remove` here. I believe this happens due to the same reason mentioned in https://github.com/gazebosim/gz-gui/pull/535#discussion_r1174074488. Crash is likely due to shared pointers in gz-rendering removing invalid resources when they go out of scope. It's hard to debug where that's coming from since it only happens on Jammy github actions and not locally. Still need to track this down